components/Movie/index.js

```diff
@@ -22,7 +22,7 @@ export default function Movie({ movie }) {
     async function fetchData() {
       try {
         const response = await fetch(
-          `https://api.themoviedb.org/3/movie/${movie.id}?api_key=657b6c5e2a2ab2cafa267e54252ca1a7&language=eng-US`
+          `https://api.themoviedb.org/3/movie/${movie.id}?api_key=${process.env.NEXT_PUBLIC_API_KEY}&language=eng-US`
```

Comment:
Hopefully you changed your `API` key as well, (like delete the old own, and get a new one) Because the ´API` key is visibile in old commits ;-)

Reply:
I deleted my account and generated a new API Key 🤓